pages/home.tsx

```diff
@@ -106,20 +109,31 @@ const Home: React.FC = () => {
     return result;
   }
 
+  
   useEffect(() => {
-    fetch("http://localhost:8000/api/course/courses")
-      .then((res) => {
-        return res.json();
-      })
-      .then((data) => {
-        const chunkedArray: Course[][] = chunkArray(data.courses, 6);
-        setCourses(chunkedArray);
-      });
+    setLoading(true); 
+    /*const delay = 2000; for testing purposes only delete after*/
+      fetch("http://localhost:8000/api/course/courses")
+        .then((res) => {
+          return res.json();
+        })
+        .then((data) => {
+          const chunkedArray: Course[][] = chunkArray(data.courses, 6);
+          setCourses(chunkedArray);
+          setLoading(true); /*change to false once tested*/
+        });
+        /*.catch((error) => {
```

Comment:
please add the catch section back in, remove unwanted comments such as delay above